bin/cylc

```diff
@@ -165,6 +165,16 @@ run_cylc_command() {
     exec "${CYLC_HOME_BIN}/cylc-${UTIL}" "$@"
 }
 
+pager () {
+    if [[ -n $PAGER ]]; then
+        echo "$PAGER"
+    elif command -v less; then
```

Comment:
```suggestion
    elif command -v less > /dev/null; then
```

This line prints to stdout, resulting (with the next line) in (e.g.) `/usr/bin/less less`